Description: FTBFS on hurd-i386
 Hello,
 .
 wmrack currently FTBFS on hurd-i386, due to inclusion of a linux-specific
 header <linux/cdrom.h>. The attached patch fixes it by including
 <sys/cdrom.h> instead, could you please apply it?
 .
 Thanks,
 Samuel
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=647977
Author: Samuel Thibault <sthibault@debian.org>
Reviewed-By: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2014-08-14

--- a/cdrom.c
+++ b/cdrom.c
@@ -24,6 +24,9 @@
 #ifdef linux
 #  include <sys/vfs.h>
 #  include <linux/cdrom.h>
+#elif defined(__GNU__)
+#  include <sys/cdrom.h>
+#  include <sys/statfs.h>
 #elif defined(__sun) && defined(__SVR4) /* Solaris */
 #  include <sys/cdio.h>
 #  include <sys/statvfs.h>
